spec/lib/movement_api/shapefile_importers/congressional_district_importer_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+require "movement_api/shapefile_importers/congressional_district_importer"
+
+module MovementApi
+  module ShapefileImporters
+    describe CongressionalDistrictImporter do
+      let(:file) { "spec/support/fixtures/shapefiles/congressional_districts/file.shp" }
+
+      subject { CongressionalDistrictImporter.new(file) }
+
+      it "has the right number of districts in the shapefile" do
+        RGeo::Shapefile::Reader.open('spec/support/fixtures/shapefiles/congressional_districts/file.shp', srid: 4326) do |file|
+          expect(file.num_records).to eq 4
+        end
+      end
+
+      describe "#call" do
+        it "creates CongressionalDistrict models" do
+          expect { subject.call }.to change{CongressionalDistrict.count}.from(0).to(3)
+        end
+      end
```

Comment:
I'm not sure what else we can spec for here? How difficult is it to create new shapefiles? The only thing I can think of regarding specs is to have multiple files with different content to account for geometry, CD114FP codes, etc.

If creating the files is too difficult, we could create mocks and pass them as a return value into `RGeo::Shapefile::Reader.open`